### PR TITLE
Tg feature refetchdata

### PR DESCRIPTION
**Pull request by talgrewal**
# TG Feature Refetchdata

- Change mutation on Unsubscribe and Subscribe button
- Add Subscription to Campaign Container to get real time campaign data
- Add Navigation to confirmation screen when unsubscribe and subscribe mutation triggered
- Update number of subscribers for campaign based on current users' actions

### CODE REVIEW

File: client/js/context/UserProvider.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            }
+          }}
+        </Query>
+      )
+    );
+  }
+}
+
+export {UserContext};
+export default UserProvider;
```
Comment (siddharthparmar7) [inline client/js/context/UserProvider.js]: Please delete this file.

File: client/js/screens/Home/Home.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const MOCK_DATA = [
 const Home = ({navigation}) => {
   return (
     <>
-      <View style={styles.carouselContainer}>
+      <View height={'22%'} style={styles.carouselContainer}>
```
Comment (siddharthparmar7) [inline client/js/screens/Home/Home.js]: Please remove all the inline styles and move them to styles.js file.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts and remove all the references to the user context to avoid errors related to user context.

File: client/js/screens/Home/Home.js (hunk 2 above)
Comment (talgrewal) [reply client/js/screens/Home/Home.js]: Done

File: client/js/context/UserProvider.js (hunk 1 above)
Comment (talgrewal) [reply client/js/context/UserProvider.js]: done.

**Comment by talgrewal** [comment]
Ready for merge.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**